tls/s2n_server_extensions.c

```diff
@@ -191,3 +203,14 @@ int s2n_recv_server_sct_list(struct s2n_connection *conn, struct s2n_stuffer *ex
 
     return 0;
 }
+
+int s2n_recv_server_max_frag_len(struct s2n_connection *conn, struct s2n_stuffer *extension)
+{
+    uint16_t max_fragment_length;
+    GUARD(s2n_stuffer_read_uint16(extension, &max_fragment_length));
```

Comment:
Thank you for your pointers. 
I have updated PR to use uint8_t at all steps during TLS handshake.